### PR TITLE
Add beta channel to SDK archive

### PR DESCRIPTION
**Pull request by mit-mit**
@athomas @kevmoo this sorta works; I do see the beta channel when I now `webdev serve` this. However, for reasons I do not get I also see a build versioned `2.8.0-dev.20.7` under beta !?

Screenshot 2020-04-21 at 19 37 08

Screenshot 2020-04-21 at 19 37 02

### CODE REVIEW

**Comment by mit-mit** [comment]
@kwalrath fyi

**Comment by athomas** [comment]
Re: "2.8.0-dev.20.7 under beta !?"

That's a beta build using the Phase I version scheme. I think it was a mistake to publish this so I just removed it from the release directory on GCP.

**Comment by kevmoo** [comment]
...don't forget to rebuild!

**Comment by mit-mit** [comment]
Working now, staged here: https://mit-staging.web.app/tools/sdk/archive

**Comment by mit-mit** [comment]
@kwalrath can you please review the verbiage in `src/tools/sdk/archive/index.md` ?

File: src/tools/sdk/archive/index.md [hunk 1]
```diff
@@ -28,11 +28,19 @@ Stable channel builds are tested and approved for production use.
 
 {% include_relative _archives_table.html channel="stable" %}
 
+## Beta channel
+
+Beta channel builds are preview builds for the stable channel.
+We recommend testing, but not releasing, your apps against beta
+to preview new features, or test compatibility with future releases.
```
Comment (kwalrath) [inline src/tools/sdk/archive/index.md:35]:
```suggestion
to preview new features or test compatibility with future releases.
```

Comment (kwalrath) [reply src/tools/sdk/archive/index.md:35]: Should we add anything like "Beta channel builds are not suitable for production use." (to match dev channel)?

Comment (mit-mit) [reply src/tools/sdk/archive/index.md]: Yup, done and done

**Pull request merged**